### ISSUE TITLE
Image height and width is not getting rendered

### ISSUE THREAD

**Issue by DenverCoder1** [nadar/quill-delta-parser#70]
**Describe the bug**

When the height and width attributes appear in an image, they do not get rendered in the HTML.

**The delta code which generates the Problem**

Example:
```json
{"ops":[{"attributes":{"height":"141","width":"210","color":"#000000","background":"transparent"},"insert":{"image":"https://lh6.googleusercontent.com/XhTtDRQlxIzcbEk-lQ38XDV0p4Gg7A0vYrP-r6huUZXyoGDMdKcb7jkl5i31BfQPGAhK3Yk_AEUrw1e8fEdCxWLVCl2UCzPHoAXDg4HqK_KlUGH4IdnZKyPqVLkyCm5PFFJ-AAEEclxnQX7jvnxm7aw"}},{"insert":"\n"}]}
```

**The expected html output the delta should produce**

Example (Expected)

```html
<p><span style="color:#000000"><img src="https://lh6.googleusercontent.com/XhTtDRQlxIzcbEk-lQ38XDV0p4Gg7A0vYrP-r6huUZXyoGDMdKcb7jkl5i31BfQPGAhK3Yk_AEUrw1e8fEdCxWLVCl2UCzPHoAXDg4HqK_KlUGH4IdnZKyPqVLkyCm5PFFJ-AAEEclxnQX7jvnxm7aw" height="141px" width="210px" alt="" class="img-responsive img-fluid" /></span></p>
```

Actual:
```html
<p><span style="color:#000000"><img src="https://lh6.googleusercontent.com/XhTtDRQlxIzcbEk-lQ38XDV0p4Gg7A0vYrP-r6huUZXyoGDMdKcb7jkl5i31BfQPGAhK3Yk_AEUrw1e8fEdCxWLVCl2UCzPHoAXDg4HqK_KlUGH4IdnZKyPqVLkyCm5PFFJ-AAEEclxnQX7jvnxm7aw" alt="" class="img-responsive img-fluid" /></span></p>
```

**Expected behavior**
`height="141px" width="210px"` should appear when `"height":"141","width":"210"` appears in the attributes.

**Screenshots**
Actual:
![image](https://user-images.githubusercontent.com/20955511/183215549-a96d0a16-510f-46f1-b0a1-49dd804adf29.png)
Expected:
![image](https://user-images.githubusercontent.com/20955511/183215608-8423d8df-2798-4d71-9d6a-199be83f850d.png)

**Additional context**
These attributes appear in quill after pasting from a Google doc with a resized image.

**Comment by nadar** [comment]
👍 i will take a look asap

**Comment by DenverCoder1** [comment]
Thanks for the quick fix!

**Comment by nadar** [comment]
Welcome, its no released s version 3.2.0
